scannergradle.properties

```
@@ -9,10 +9,10 @@ sourcesUrl=https://github.com/SonarSource/sonar-scanner-gradle
 archivedVersions=2.8,3.0,3.1,3.1.1,3.2,3.3,3.4.0.2513,3.5.0.2730,4.0.0.2929
 publicVersions=4.2.1.3168
```

Let's change directly the public version. It will be visible for users only when PM deploys the Update Center (next Monday, the 4th).
```suggestion
archivedVersions=2.8,3.0,3.1,3.1.1,3.2,3.3,3.4.0.2513,3.5.0.2730,4.0.0.2929,4.2.1.3168
publicVersions=4.3.1.3277 
```